packages/itwinui-react/src/core/ButtonGroup/ButtonGroup.tsx

```diff
@@ -8,6 +8,16 @@ import { useOverflow, useMergedRefs, Box } from '../utils/index.js';
 import type { PolymorphicForwardRefComponent } from '../utils/index.js';
 import { FloatingDelayGroup } from '@floating-ui/react';
 
+// ----------------------------------------------------------------------------
+
+/** This context is used for letting descendant IconButtons know the ButtonGroup's orientation. */
+export const ButtonGroupContext = React.createContext<string | undefined>(
```

Comment:
Do we want to support allowing users to set the `labelProps` of all `IconButton` descendants by setting it just once in the `ButtonGroup`? Or would that be too specific and thus better to not support and thus not clutter the `ButtonGroup`'s available props?

Reply:
No, this part is not controllable by the user. All we're doing here with the context is changing the default behavior of IconButton.

The user can always create a wrapper component if they want multiple IconButtons to behave similarly.